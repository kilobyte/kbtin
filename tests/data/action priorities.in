#act {aaa%0} {#showme act: a} {a}
#act {aaa%1} {#showme act: 0} {0}
#act {aaa%2} {#showme act: 0.09} {0.09}
#act {aaa%3} {#showme act: 0.1} {0.1}
#act {aaa%4} {#showme act: 0.10} {0.10}
#act {aaa%5} {#showme act: 0.19} {0.19}
#act {aaa%6} {#showme act: 0a.1} {0a.1}
#act {aaa%7} {#showme act: a06z} {a06z}
#act {aaa%8} {#showme act: a6x} {a6x}
#act {aaa%9} {#showme act: 0.2} {0.2}
#doact aaa
