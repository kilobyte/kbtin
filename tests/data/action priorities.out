act: 0
act: 0.1
act: 0.2
act: 0.09
act: 0.10
act: 0.19
act: 0a.1
act: a
act: a6x
act: a06z
